### DISCUSSION TITLE
How do I write detection logs to a text file?

### DISCUSSION THREAD

**Discussion by ravesmith** [shubham0204/FaceRecognition_With_FaceNet_Android#22] [Q&A]
I apologize for asking something so basic.

How can I save in a local text file the name of the faces that the application has detected?

Thank you very much for your help.

**Comment by shubham0204** [comment]
You can actually print the logcat text in Android Studio. Select the appropriate log level ( most probably INFO level ) and run the app. You can then click the **Print** icon in the logcat. See [View your app logs](https://developer.android.com/studio/debug/am-logcat#running).
